scripts/create_newcase

```diff
@@ -149,6 +155,9 @@ OR
         expect(args.gridfile is not None,
                "User grid specification file must be set if the user grid is requested")
 
+    expect(not (int(args.ncouplers) > 1 and int(args.ninst) > 1),
```

Comment:
This needs to change to address the above comment - either --ninst or --ncouplers should be allowed.